bin/monitoring

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
```

Comment:
I'd put this in udmis/bin -- just trying to push some things down to more specific levels when we can.